### PR TITLE
Adds terraform documentation

### PR DESCRIPTION
**Pull request by mdesanti**
## Summary

Adds readme on how to create infrastructure with terraform.

### CODE REVIEW

File: infrastructure/docs/aws/docs/terraform.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Visit [this page](https://github.com/Wolox/terraform-base-infra#elasticbeanstalk--rds) to find an example template.
+
+### S3 Website
+
+Visit [this page](https://github.com/Wolox/terraform-base-infra#s3-website) to find an example template.
+
+## Apply changes
+
+Once the `.tf` file is complete and the default values were replaced, run `terraform plan`. This will generate a plan an list all the resources to be created. If you're ok with this, run `terraform apply`. This will re-run the plan and promp for your confirmation to create the infrastructure. Type `yes` and let Terraform do it's job
```
Comment (Gmarn) [inline infrastructure/docs/aws/docs/terraform.md]: There is no need to run `terraform init` ?

**Comment by gussiciliano** [comment]
@Gmarn @mdesanti ping here

**Comment by gussiciliano** [comment]
Also, please follow the PR naming convention described in #90 :)

**Comment by gussiciliano** [comment]
@Gmarn @mdesanti ping here

**Pull request merged**